api/backup.go

```diff
@@ -90,3 +90,13 @@ func (s *Server) BackupDelete(w http.ResponseWriter, req *http.Request) error {
 	logrus.Debugf("Removed backup %v of volume %v", input.Name, volName)
 	return nil
 }
+
+func (s *Server) BackupStatus(w http.ResponseWriter, req *http.Request) error {
```

Comment:
Don't need an extra API. It should be exposed as a field in the volume object. https://github.com/longhorn/longhorn-manager/blob/master/api/model.go#L482